recipes/tryingsnake/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "tryingsnake" %}
+{% set version = "0.4.0.dev1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 997c0997471c86f78149c02bf4024cbf82a9559472548ae393d9207a6f678405
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - tryingsnake
+  requires:
```

Comment:
Since you are just testing for the import, I don't think you need the `requires` section and `pytest`. Can you please try removing that?